submissions/oluwaboriShobayo/HTML/Laptop media query.css

```diff
@@ -0,0 +1,197 @@
+@media  only screen and (min-device-width: 1200px) {
```

Comment:
What is this CSS style for? Meanwhile, all CSS file should be inside your CSS folder